setup.py

```diff
@@ -82,13 +82,19 @@ def get_version(version_file):
     ],
     python_requires=">=3.6",
     # What does your project relate to?
-    keywords="skeleton",
+    keywords="li-pca",
     packages=find_packages(where="src"),
     package_dir={"": "src"},
-    package_data={"example": ["data/*.txt"]},
+    package_data={"api": ["data/*.txt"]},
     py_modules=[splitext(basename(path))[0] for path in glob("src/*.py")],
     include_package_data=True,
-    install_requires=["docopt", "schema", "setuptools >= 24.2.0"],
+    install_requires=[
+        "docopt",
+        "schema",
+        "setuptools >= 24.2.0",
+        "connexion == 2.6.0",
+        "python_dateutil == 2.6.0",
```

Comment:
These should be alphabetized:
```suggestion
        "connexion == 2.6.0",
        "docopt",
        "python_dateutil == 2.6.0",
        "schema",
        "setuptools >= 24.2.0",
```